HACKING.md

```diff
@@ -61,6 +61,11 @@ itself as third word (cf. the [documentation for `expected_region_highlight`]
       "1 8 fg=white,bold,bg=red" # rm -rf /
     )
 
+Testing the `regexp` highlighter
```

Comment:
Perhaps instead of making a new section, rename the section above

    Testing the `pattern` and `regex` highlighters

and use

    Because the `pattern` and `regex` highlighters modify

after `pattern` in the body.